frame/evm/src/backend.rs

```diff
@@ -141,93 +141,3 @@ impl<'vicinity, T: Trait> BackendT for Backend<'vicinity, T> {
 		AccountStorages::get(address, index)
 	}
 }
-
-impl<'vicinity, T: Trait> ApplyBackend for Backend<'vicinity, T> {
```

Comment:
Those are just leftover code that is not needed any more after unified EVM runner.